capstone-frontend/src/App.jsx

```diff
@@ -20,7 +22,16 @@ function App() {
         console.log("Server connected | WS");
       });
       socket.current.addEventListener("message", function (event) {
-        console.log("Server response", event.data);
+        if(/\d/.test(event.data)) {
+          const str = event.data;
+          const numbers = str.match(/\d+/g);
+          console.log("Server response", numbers);
```

Comment:
What are these numbers for/what's the purpose of this check? Used later?

Reply:
This test checks to see if the message I received includes numbers => meaning this is an emergency alert because I include the userId which is an Int in this message. This helped me differentiate the messages I was getting when I refreshed the server from ones that actually should prompt the modal.

Reply:
You can send JSON data in the websocket and then you can use `JSON.parse` to parse it, and you won't have to manually use regex to get information out of it